packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -10,6 +10,11 @@ UserMessages get userMessages => context.get<UserMessages>();
 
 /// Class containing message strings that can be produced by Flutter tools.
 class UserMessages {
+  // Common messages.
```

Comment:
Does "common messages" mean anything as a comment?

Reply:
All of the other messages in this file are preceded by "Messages used in X", so I was trying to clarify this is a message used in multiple components.